spec/features/dashboard_spec.rb

```diff
@@ -36,7 +36,7 @@
       sign_in_as(user)
       visit dashboard_path
 
-      click_on 'All Open To Help'
+      click_on 'Open To Help'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.